src/test/java/soolutions/hometaskapp/account/TransferTests.java

```diff
@@ -0,0 +1,77 @@
+package soolutions.hometaskapp.account;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import soolutions.hometaskapp.account.Account;
+import soolutions.hometaskapp.account.BasicAccount;
+import soolutions.hometaskapp.account.Deposit;
+import soolutions.hometaskapp.account.Transaction;
+import soolutions.hometaskapp.account.Transfer;
+import soolutions.hometaskapp.account.Transfer.InvalidTransactionException;
+import soolutions.hometaskapp.common.Amount;
+import soolutions.hometaskapp.user.BasicUser;
+
+public class TransferTests {
+    private AccountFactory dollarAccountFactory;
+
+    @BeforeEach
+    void setup() {
+        dollarAccountFactory = new AccountFactory("USD");
+    }
+
+    @Test
+    void transferShouldFailForClosedAccounts() {
```

Comment:
Should we add a similar test for case, when `to` account is closed?

Reply:
Sure, but the issue is none of my tests are running, can you please check?

Reply:
As I see, there is a problem in `@BeforeEach`: you didn't imported `import org.junit.jupiter.api.BeforeEach;`.

Also there is another problem in `AccountFactory`. I'll mention it.